acvm/src/pwg.rs

```diff
@@ -213,6 +247,13 @@ fn insert_value(
     Ok(())
 }
 
+#[derive(Debug, PartialEq, Clone)]
+pub struct UnresolvedBrillig {
+    pub brillig: Brillig,
+    // Information for, if there is a pending foreign call/oracle
```

Comment:
Seems like `UnresolvedBrillig` can only exist if there is a foreign call so this comment is misleading.

```suggestion
    /// Inputs for a pending foreign call required to restart bytecode processing.
```